File: 0088-merge-sorted-array/0088-merge-sorted-array.py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        k = m
        i = 0
        while k < m + n:
            nums1[k] = nums2[i]
            k += 1
            i += 1
        nums1.sort()      

File: 0088-merge-sorted-array/0088-merge-sorted-array.js
/**
 * @param {number[]} nums1
 * @param {number} m
 * @param {number[]} nums2
 * @param {number} n
 * @return {void} Do not return anything, modify nums1 in-place instead.
 */
var merge = function(nums1, m, nums2, n) {
    let x = 0;
    for(let i = m; i < (m + n); i++) {
        nums1[i] = nums2[x];
        x++;
    }
    nums1.sort((x, y) => x - y);
    return;
};